explorer/src/pages/AccountDetailsPage.tsx

```diff
@@ -184,13 +199,18 @@ export function AccountHeader({
 
   if (isMetaplexNFT(data, mintInfo)) {
     return (
-      <NFTHeader
+      <MetaplexNFTHeader
         nftData={(data as TokenProgramData).nftData!}
         address={address}
       />
     );
   }
 
+  const nftokenNFT = isNFTokenAccount(account);
+  if (nftokenNFT && account) {
```

Comment:
Make `account` required in `isNFTokenAccount()` and then null check it everywhere it's used. More efficient since `isNFTokenAccount()` needs to do less work, and you don't even call it if account is `null`.

```typescript
if (account && isNFTokenAccount(account)) {
  /* ... * /
}
```